### PR TITLE
Kuma Report, April 2017

### PR DESCRIPTION
**Pull request by jwhitlock**
Another month, another report. People that may want to check what I say about them for accuracy:

* @jgmize and @metadave - AWS work this month
* @schalkneethling - Embedded sample work next month
* @escattone and @stephaniehobson - Gross mischaracterizations of your work and plans

Of course, we can always push updates if I get something obviously wrong.

### CODE REVIEW

**Comment by metadave** [comment]
👍

**Comment by escattone** [comment]
👍 As always, well written and well done. Thanks @jwhitlock !

File: _posts/2017-05-02-kuma-report.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+the engine of
+[MDN](https://developer.mozilla.org):
+
+- Explored faster paths to AWS
+- Maintained quality with new robots
+- Improved the macros dashboard
+- Goodbye BrowserCompat API, Hello Browser Compat Data
+- Shipped tweaks and fixes
+
+Here's the plan for April:
```
Comment (stephaniehobson) [inline _posts/2017-05-02-kuma-report.md]: May?

**Comment by stephaniehobson** [comment]
No gross mischaracterizations 🚢

File: _posts/2017-05-02-kuma-report.md (hunk 1 above)
Comment (jwhitlock) [inline _posts/2017-05-02-kuma-report.md]: Good catch!

**Comment by jwhitlock** [comment]
Thanks for the feedback everyone!

**Pull request merged**